src/test/java/bio/terra/service/filedata/google/firestore/TestFirestoreProvider.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.service.filedata.google.firestore;
+
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.FirestoreOptions;
+
+public final class TestFirestoreProvider {
+
+    private TestFirestoreProvider() {}
+
+    public static Firestore getFirestore() {
+        return FirestoreOptions.getDefaultInstance()
+            .toBuilder()
+            .setProjectId(System.getenv("GOOGLE_CLOUD_DATA_PROJECT"))
```

Comment:
What happens if the `GOOGLE_CLOUD_DATA_PROJECT` isn't defined?

Reply:
It will get get an empty value, try to auth with that, then fail. Not sure if that's something we have to catch since its explicitly said in the setup docs to set the variable.

Reply:
Good point! If it's a very cryptic error message, catching it explicitly could be helpful here but I'll leave that up to you!